clang/test/CheckedCRewriter/checkedregioninterweave.c

```diff
@@ -0,0 +1,25 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
```

Comment:
For these tests you want to make sure that `clang` will compile the final result, too. Look at Shilpa's tests for that.

Reply:
(By "these tests" I mean all of the ones that don't involve `-alltypes`.)